scripts/generateIconEnum.ts

```diff
@@ -7,7 +7,7 @@ const ICON_ENUM_OUTPUT_PATH = './src/foundation/Icon/IconEnum.ts';
 
 (async () => {
     const iconFilePath = await fastGlob('./src/foundation/Icon/Generated/*.tsx', { objectMode: true });
-    const iconNames = iconFilePath.map((path) => path.name.replace('Icon', '').replace('.tsx', ''));
+    const iconNames = iconFilePath.map((path) => path.name.replace(/^Icon|\.tsx$/g, ''));
```

Comment:
Sonarcloud is complaining about the regex, this should fix `/(^Icon)|(\.tsx$)/g`